package.json

```diff
@@ -155,20 +156,23 @@
     "bundle-loader": "0.5.5",
     "chai": "^4.0.2",
     "coveralls": "^2.13.1",
+    "enzyme": "3.1.0",
+    "enzyme-adapter-react-15": "1.0.2",
     "es6-promise": "^4.1.0",
     "eslint": "^3.17.0",
     "eslint-plugin-babel": "^4.1.1",
     "eslint-plugin-react": "^6.10.0",
     "extract-text-webpack-plugin": "^2.0.0",
     "husky": "^0.13.2",
-    "istanbul": "0.4.5",
+    "istanbul": "1.0.0-alpha.2",
```

Comment:
Why do you use an alpha version?

Reply:
This issue: https://github.com/gotwarlost/istanbul/issues/262
Although the coverage results still seem to be inaccurate

Reply:
Definitely. If it stays this inaccurate, it's better to deactivate coverage generation and upload to coveralls as it will result in a false impression about the project.